thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/model/merge/MinMaxThresholdMergeModel.java

```diff
@@ -0,0 +1,67 @@
+package com.linkedin.thirdeye.anomalydetection.model.merge;
```

Comment:
Merged anomaly are one of the inputs of Update() method. The updated results will be written back. The input AnomalyDetectionContext should provide the necessary time series in order to update the given merged anomaly.